apps/web/src/app/(dashboard)/documents/upload-document.tsx

```diff
@@ -18,10 +19,12 @@ import { useToast } from '@documenso/ui/primitives/use-toast';
 
 export type UploadDocumentProps = {
   className?: string;
+  userId?: number;
 };
 
-export const UploadDocument = ({ className }: UploadDocumentProps) => {
+export const UploadDocument = ({ className, userId }: UploadDocumentProps) => {
```

Comment:
We could just `useSession` instead of prop drilling?

Reply:
Sure, I will update it